src/blocks/hero/components/edit.js

```diff
@@ -162,6 +163,11 @@ class Edit extends Component {
 					className={ classes }
 				>
 					<div className={ innerClasses } style={ innerStyles } >
+						{ backgroundType == 'video' ? 
+							<div className="coblocks-video-background">
+								<video playsinline="" autoplay="" muted="" loop="" src={ backgroundImg } ></video>
```

Comment:
@phpbits What do you think about making `autoplay`, `muted`, and `loop` ToggleControl options within the Background Settings panel (only if the backgroundType is 'video').

Reply:
On second though, just muted and loop - not autoplay (as it should autoplay).